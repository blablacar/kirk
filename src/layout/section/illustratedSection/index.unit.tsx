import React from 'react'
import renderer from 'react-test-renderer'

import IllustratedSection from './index'

const defaultProps = {
  illustrationUrl: 'http://blablacar.com/pic.jpg',
}

describe('IllustratedSection', () => {
  it('should render default MediaContentSection section', () => {
    const section = <IllustratedSection {...defaultProps}>Hello World</IllustratedSection>
    const renderedSection = renderer.create(section).toJSON()
    expect(renderedSection).toMatchSnapshot()
  })
})
