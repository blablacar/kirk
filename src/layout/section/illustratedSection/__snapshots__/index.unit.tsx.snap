// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IllustratedSection should render default MediaContentSection section 1`] = `
.c1 .section-content {
  padding-left: 24px;
  padding-right: 24px;
}

.c0 .section-content {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 .kirk-illustratedSection-illustration {
  margin-left: -24px;
  margin-right: -24px;
  text-align: center;
}

.c0 .kirk-illustratedSection-illustration img {
  max-width: 100%;
}

@media (min-width:800px) {
  .c1 .section-content {
    margin-left: auto;
    margin-right: auto;
    max-width: 662px;
  }

  .c1 .section-content.section-content--large {
    max-width: 1016px;
  }
}

@media (min-width:800px) {
  .c0 .section-content {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }

  .c0 .kirk-illustratedSection-illustration {
    margin-left: 0;
    margin-right: 0;
    width: 33.33%;
  }

  .c0 .kirk-illustratedSection-content {
    margin-left: 24px;
    width: 66.66%;
  }
}

<div
  className="c0 c1"
  role="presentation"
>
  <article
    className="section-content section-content--large"
  >
    <div
      className="kirk-illustratedSection-illustration"
    >
      <img
        alt=""
        src="http://blablacar.com/pic.jpg"
      />
    </div>
    <div
      className="kirk-illustratedSection-content"
    >
      Hello World
    </div>
  </article>
</div>
`;
